Assets/HoloToolkit/Input/Scripts/InputSources/EditorInputSource.cs

```diff
@@ -49,7 +49,7 @@ public ButtonStates()
 
         public bool SupportsPosition;
         public bool SupportsRotation;
-        public bool SupportsPointing;
```

Comment:
The name change is to better match the platform terminology. There's both a "pointer ray" and a "grip ray".